cli/config/configuration-qemu.toml.in

```diff
@@ -222,8 +222,8 @@ enable_iothreads = @DEFENABLEIOTHREADS@
 # Use this parameter when using some large PCI bar devices, such as Nvidia GPU
 # The value means the number of pcie_root_port
 # This value is valid for "q35" machine type
-# Default 1
-pcie_root_port = 1
+# Default 2
+pcie_root_port = 2
```

Comment:
this was introduced in a previous commit, please squash it

Reply:
Done